stories/text/text.stories.js

```diff
@@ -0,0 +1,167 @@
+    Dolor magna eget est lorem ipsum dolor sit. Aliquam nulla facilisi cras fermentum odio eu feugiat pretium nibh.
+    Augue neque gravida in fermentum et. Proin fermentum leo vel orci porta non pulvinar.</p>
+`;
+
+maxLines.storyName = 'Max lines';
+maxLines.parameters = {
+    docs: {
+        iframeHeight: 300,
+        storyDescription: `The text component can be displayed with a maximum number of lines.
+When the maximum is reached, the text truncates and displays an ellipsis. To display text with a maximum line count,
+add the <code class="docs-code">fd-text--max-lines</code> modifier class and an inline style rule with the number of
+lines to the main element. For example, add <code class="docs-code">style="-webkit-line-clamp: 3;"</code> to display
+three lines of text.
+
+**Note**: Property <code class="docs-code">-webkit-line-clamp</code> doesn't work in IE11 and should be changed
+to <code class="docs-code">height</code>. For example, <code class="docs-code">style="height: 200px;"</code> 
+`
+    }
+};
+
+export const hyphenation = () => `<div class="example-container">
+        <div class="fddocs-text">
+            <h3>Without hyphens</h3>
+            <p class="fd-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
+            ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore
+            eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia
+            deserunt mollit anim id est laborum.</p>
+        </div>
+
+        <div class="fddocs-text">
+            <h3>Hyphenation</h3>
+            <p class="fd-text fd-text--hyphenation">Lorem ipsum dolor sit amet, con&shy;sectetur adip&shy;iscing el&shy;it, sed do eiusmod tempor
+            incididunt ut labore et do&shy;lore magna aliqua. Ut enim ad mi&shy;nim veniam, quis nostrud exer&shy;citation ullamco laboris nisi ut
+            aliquip ex ea commodo conse&shy;quat. Duis aute irure dolor in repre&shy;henderit in vo&shy;luptate velit esse cillum dolore
+            eu fu&shy;giat nulla pari&shy;atur. Excepteur sint occaecat cupidatat non pro&shy;ident, sunt in culpa qui officia
+            deserunt mollit anim id est la&shy;bo&shy;rum.</p>
+        </div>
+    </div>
+`;
+
+hyphenation.storyName = 'Hyphenation';
+hyphenation.parameters = {
+    docs: {
+        iframeHeight: 200,
+        storyDescription: `The text component can display words that are broken at appropriate hyphenation
+points in a text block. To display hyphens, add the <code class="docs-code">fd-text--hyphenation</code> to the main element.
+        
+It is also possible to suggest line break opportunities with two Unicode characters that manually specify
```

Comment:
Oops my bad here, all sentences with a colon ":" should be bolded. So just bold "It is also possible to suggest line break opportunities with two Unicode characters that manually specify potential line breakpoints:"